query.go

```diff
@@ -235,6 +228,33 @@ func (iter Enumerator) Select(transform Transform) Enumerator {
 	return retval
 }
 
+type selectManyer struct {
```

Comment:
`selectManyer` took me a second to realize it wasn't a typo. Meh, it's internal.